config/runtime/server.go

```diff
@@ -119,14 +134,28 @@ func NewServerConfiguration(conf *config.Couper, log *logrus.Entry) (ServerConfi
 		}
 
 		if srvConf.Files != nil {
+			corsOptions, err := middleware.NewCORSOptions(
+				getCORS(srvConf.CORS, srvConf.Files.CORS),
+			)
+			if err != nil {
+				return nil, err
+			}
+
 			fileHandler, err := handler.NewFile(serverOptions.FileBasePath, srvConf.Files.DocumentRoot, serverOptions)
 			if err != nil {
 				return nil, err
 			}
 
+			var h http.Handler
+			if corsOptions != nil {
```

Comment:
`NewCORSOptions` can be moved to this line so we have related code in one place (and save some lines).

Reply:
The same applies to `SPA` above.